bundles/framework/publisher2/tools/GetInfoTool.js

```diff
@@ -103,6 +105,9 @@ Oskari.clazz.define('Oskari.mapframework.publisher.tool.GetInfoTool',
                             me.values.colourScheme = plugin.config.colourScheme;
                             me._sendColourSchemeChangedEvent(me.values.colourScheme);
                         }
+                        if (plugin.config && plugin.config.noUI) {
+                            me.noUiIsCheckedInModifyMode = !!plugin.config.noUI;
```

Comment:
We might as well use the same variable for this? Makes it easier to read.
```suggestion
                            me.noUi = !!plugin.config.noUI;
```